lib/connectwise-ruby-sdk/models/ticket.rb

```diff
@@ -851,8 +851,8 @@ def list_invalid_properties
         invalid_properties.push("invalid value for 'po_number', the character length must be smaller than or equal to 50.")
       end
 
-      if !@automatic_email_cc.nil? && @automatic_email_cc.to_s.length > 1000
-        invalid_properties.push("invalid value for 'automatic_email_cc', the character length must be smaller than or equal to 1000.")
+      if !@automatic_email_cc.nil? && @automatic_email_cc.to_s.length > 10000
```

Comment:
Can this be made a constant somewhere earlier in the file?

Reply:
yes please move this to a constant at the top of the class or something, like `MAX_EMAIL_LENGTH=10_000`